contrib/rename_image.bzl

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 # Migrated from https://github.com/GoogleContainerTools/base-images-docker/blob/4f2fc8da248a61c3f8e13bbb43e9db6c0ed44ba3/util/run.bzl#L264
+"""A rule to rename the image from a <lang>_image or container_image target.
+"""
```

Comment:
Can you also put the "Migrated from ..." to be in the body of the docstring?

Reply:
Done